Classes/Common/MetsDocument.php

```diff
@@ -1508,7 +1508,7 @@ private function getPhysicalElements(array $elementNodes, array $fileUse): array
                 }
             }
 
-            // Get track info wtih begin end extent time for later assignment with musical
+            // Get track info with begin end extent time for later assignment with musical
```

Comment:
I am not sure here. Perhaps "and extent time" was intended.

Reply:
Yes, it should be "and" instead of "end".
```suggestion
            // Get track info with begin and extent time for later assignment with musical
```